text-generation-inference/docker/entrypoint.sh

```diff
@@ -10,6 +10,19 @@ if [[ -z "${MAX_BATCH_SIZE}" ]]; then
 fi
 export MAX_BATCH_SIZE="${MAX_BATCH_SIZE}"
 
+# At some point we used to have MAX_INPUT_LENGTH, now we should use MAX_INPUT_TOKENS
+# (This would be done automatically by the launcher, but we need to calculate the
+# MAX_BATCH_PREFILL_TOKENS if not set)
+if [[ -z "${MAX_INPUT_TOKENS}" && -n ${MAX_INPUT_LENGTH} ]]; then
+  MAX_INPUT_TOKENS=${MAX_INPUT_LENGTH}
+  unset MAX_INPUT_LENGTH
+fi
```

Comment:
```suggestion
if [[ -z "${MAX_INPUT_TOKENS}" && -n ${MAX_INPUT_LENGTH} ]]; then
  MAX_INPUT_TOKENS=${MAX_INPUT_LENGTH}
fi

unset MAX_INPUT_LENGTH
```

Maybe we should unset MAX_INPUT_LENGTH anyway to prevent any downstream use of it as it is deprecated